### PR TITLE
cart feature

### PR DESCRIPTION
**Pull request by anupkgurung**
cart

### CODE REVIEW

File: style/wishlist.css [hunk 1]
```diff
@@ -1,7 +1,8 @@
 .authentication {
     display: flex;
 }
-.whishlist-container {
+.whishlist-container ,
+.cart-container {
     margin: 5rem 0rem;
     padding: 5rem 0rem;
 }
```
Comment (sanket4120) [inline style/wishlist.css:8]: Why are you creating separate classes for applying flex, giving margin & padding
Are you not using component library

Comment (anupkgurung) [reply style/wishlist.css:8]: it is specific to the containers which are not there in component library.

**Review by sanket4120 — APPROVED** [review]
I will suggest to remove that red shadow Rest looks good

**Review by ishitakeshawani — APPROVED** [review]

**Comment by anupkgurung** [comment]
> I will suggest to remove that red shadow Rest looks good

Oh Thanks for specifying. Will have to change that in Component Library.

**Pull request merged**